assets/js/util.js

```diff
@@ -162,6 +165,8 @@ $(document).ready(function () {
         return false;
     });
 
+    $('#unobtrusiveWarning').addClass('hide');
```

Comment:
Um, polyfill? I have kept it inside `$( function() {} );` so that as soon as the page loads, the warning is hidden. It is triggered only on service call (if duration conditions satisfy!). This is similar to what I have in the `#backToTop` button!